roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -93,9 +137,20 @@
   register: existing_metrics_rc
   changed_when: no
 
+- command: >
+    {{openshift_client_binary}}
+    --config={{mktemp.stdout}}/admin.kubeconfig
+    get deployment
+    -l k8s-app=metrics-server
+    -o name
+    -n {{openshift_metrics_server_project}}
+  register: existing_metrics_server_rc
```

Comment:
this variable doesn't make sense -- you're getting a deployment, but it's called `_rc`